tests/monitoring/component_monitoring.go

```diff
@@ -127,14 +122,7 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 
 		AfterEach(func() {
 			scales.RestoreAllScales()
-
-			time.Sleep(10 * time.Second)
-			alerts := []string{
-				virtOperator.downAlert, virtOperator.noReadyAlert, virtOperator.lowCountAlert,
-				virtController.downAlert, virtController.noReadyAlert, virtController.lowCountAlert,
-				virtApi.downAlert, virtApi.noReadyAlert, virtApi.lowCountAlert,
-			}
-			libmonitoring.WaitUntilAlertDoesNotExist(virtClient, alerts...)
+			waitUntilComponentsAlertsAreGone(virtClient)
```

Comment:
nit: Gone is a bit not clear maybe DoesNotExist?

Reply:
yup, was a bit too colloquial, changed